frontend/lib/src/controllers/transaction_controller.dart

```diff
@@ -4,23 +4,37 @@ import 'package:frontend/src/models/transaction.dart';
 import 'package:frontend/src/services/transaction_service.dart';
 
 List<String> transactionTypes = ['Credit', 'Dept', 'Transfer'];
+List<String> transactionCategory = [
```

Comment:
all transaction categories must be included in the new enum

for getting the values only meant for, for example Transfer type
i tried this 
![image](https://github.com/framgia/sph-flutter/assets/95029803/97dec601-202e-4847-a391-2549f9dd9676)